src/main/java/uk/gov/hmcts/reform/draftstore/config/DraftStoreConfig.java

```diff
@@ -41,7 +43,11 @@ public Filter etagFilter() {
 
     @Bean
     public DraftStoreDAO draftDocumentDAO(NamedParameterJdbcTemplate jdbcTemplate) {
-        return new DraftStoreDAO(jdbcTemplate);
+        return new DraftStoreDAO(
+            jdbcTemplate,
+            maxStaleDaysDefault,
+            Clock.systemUTC()
```

Comment:
I'd let the time zone settings be configured at system property level (is there a way not to provide it here?). This way we would always have just one point where time zone is set.